contracts/test/TestTransferFrom.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "../shared/TransferHelper.sol";
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract TestTransferFrom {
+
+    event LogTransferHelperInputValidation1Failed(bool tokenIsContract, address from, address to);

```

Comment:
@matthiaszimmermann TODO also add these to ITreasury as well as AyiiProduct (otherwise log decoding won't work)